public/js/search_terms.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
Is it correct to have this file in the repository?
Shouldn't it be generated on build?